deploy-board/deploy_board/templates/groups/asg_policy.tmpl

```diff
@@ -455,6 +453,159 @@ aria-hidden="true">
         </div>
 
         <script>
+            function validateStepScalingConfig(stepScalingConfig) {
```

Comment:
should these js code be in a separate .js file? the template file is getting lengthy